packages/react-components/src/input/src/wrappedInputPropsAdapter.ts

```diff
@@ -0,0 +1,29 @@
+import { UseFieldInputPropsReturn, useFieldInputProps } from "../../field";
+
+export interface FieldWrappedInputProps extends Omit<UseFieldInputPropsReturn[0], "className"> {
```

Comment:
This array index thing is weird.

Looking at the code in ``FieldContext.tsx` I would cheat a little bit and do:

```
export type UseFieldInputPropsReturn = {
    validationState?: "valid" | "invalid";
    id?: string;
    required?: boolean;
    disabled?: boolean;
    fluid?: boolean;
    size?: "sm" | "md";
    className?: string;
    "aria-labelledby"?: string;
    "aria-describedby"?: string;
};

export function useFieldInputProps(): [UseFieldInputPropsReturn, boolean] {
```

Reply:
done

Reply:
done